app/forms/journeys/additional_payments_for_teaching/claim_submission_form.rb

```diff
@@ -1,36 +1,15 @@
 module Journeys
   module AdditionalPaymentsForTeaching
     class ClaimSubmissionForm < ::ClaimSubmissionBaseForm
-      ELIGIBILITY_ATTRIBUTES = [
-        [:nqt_in_academic_year_after_itt, :boolean],
-        [:employed_as_supply_teacher, :boolean],
-        [:qualification, :string],
-        [:has_entire_term_contract, :boolean],
-        [:employed_directly, :boolean],
-        [:subject_to_disciplinary_action, :boolean],
-        [:subject_to_formal_performance_action, :boolean],
-        [:eligible_itt_subject, :string],
-        [:eligible_degree_subject, :string],
-        [:teaching_subject_now, :boolean],
-        [:itt_academic_year, AcademicYear::Type.new],
-        [:current_school_id, :string], # uuid
-        [:induction_completed, :boolean],
-        [:school_somewhere_else, :boolean]
-      ]
-
-      ELIGIBILITY_ATTRIBUTES.each do |name, type|
-        attribute name, type
-      end
-
       private
 
-      def eligibility_attribute_names
-        ELIGIBILITY_ATTRIBUTES.map(&:first)
+      def journey
+        Journeys::AdditionalPaymentsForTeaching
       end
 
       def selected_claim_policy
-        if selected_policy.present?
-          "Policies::#{selected_policy}".constantize
+        if answers.selected_policy.present?
+          "Policies::#{answers.selected_policy}".constantize
         end
```

Comment:
Please change: Since this method can only ever safely use two policies' classes, we should be able to avoid using `constantize` from user input here.

```suggestion
        answers.selected_policy == "EarlyCareerPayments" ? Policies::EarlyCareerPayments : Policies::LevellingUpPremiumPayments
        end
```

We could even consider raising an exception if it's not one of the two expected values.

Reply:
Good point. I think raising if an unexpected policy makes it here is best. Have gone for a case statement as we need to support the `selected_policy` being `nil`